subsquid/src/processor.ts

```diff
@@ -16,14 +18,17 @@ processor.setDataSource({
   chain: "wss://dali.devnets.composablefinance.ninja/parachain/alice",
 });
 
-
 // TODO add event handlers for Pablo
+processor.addEventHandler('pablo.PoolCreated', async (ctx) => {
```

Comment:
Nice! Did you try with a local node somehow? Random question :)

Reply:
No, the squid and the archive is configured for dali.devnets.composablefinance.ninja for now.
But fixing some issues related to the db at the moment when processing events.

Reply:
`PoolCreated` is processed correctly now
![Screenshot from 2022-04-27 11-17-30](https://user-images.githubusercontent.com/654927/165485634-9a6f5d58-1597-44d2-9630-2b9099c93c0b.png)

Reply:
:rocket:

Reply:
Awesome :)